Sentry 5.7.2 with timber integration crashes the app when formats a message with args

### Integration

sentry-android-timber

### Build System

Gradle

### AGP Version

7.1.3

### Proguard

Disabled

### Version

5.7.2

### Steps to Reproduce

1. Add the log:
`Timber.d("test %s, %s", 1, 1);`
2. The app crashes.

Version 5.7.1 works fine. It seems like the #1974 PR causes this. The `args` is passed incorrectly - missing asterisk. 

### Expected Result

A log message:
`test 1 1`

### Actual Result

```
Caused by: java.util.MissingFormatArgumentException: Format specifier '%s'
            	at java.util.Formatter.format(Formatter.java:2522)
            	at java.util.Formatter.format(Formatter.java:2458)
            	at java.lang.String.format(String.java:2842)
            	at timber.log.Timber$Tree.formatMessage(Timber.kt:172)
            	at timber.log.Timber$Tree.prepareLog(Timber.kt:161)
            	at timber.log.Timber$Tree.d(Timber.kt:51)
            	at io.sentry.android.timber.SentryTimberTree.d(SentryTimberTree.kt:60)
            	at timber.log.Timber$Forest.d(Timber.kt:292)
            	at timber.log.Timber.d(Unknown Source:2)
            	at the.app.MainActivity.onCreate(MainActivity.java:109)
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+* Fix: Sentry Timber integration throws an exception when using args (#1986)
+
 ## 5.7.2
 
 * Fix: bring back support for `Timber.tag` ([#1974](https://github.com/getsentry/sentry-java/pull/1974))

--- a/sentry-android-timber/src/main/java/io/sentry/android/timber/SentryTimberTree.kt
+++ b/sentry-android-timber/src/main/java/io/sentry/android/timber/SentryTimberTree.kt
@@ -42,7 +42,7 @@ class SentryTimberTree(
         message: String?,
         vararg args: Any?
     ) {
-        super.v(t, message, args)
+        super.v(t, message, *args)
         logWithSentry(Log.VERBOSE, t, message, *args)
     }
 
@@ -57,7 +57,7 @@ class SentryTimberTree(
         message: String?,
         vararg args: Any?
     ) {
-        super.d(message, args)
+        super.d(message, *args)
         logWithSentry(Log.DEBUG, null, message, *args)
     }
 
@@ -67,7 +67,7 @@ class SentryTimberTree(
         message: String?,
         vararg args: Any?
     ) {
-        super.d(t, message, args)
+        super.d(t, message, *args)
         logWithSentry(Log.DEBUG, t, message, *args)
     }
 
@@ -82,7 +82,7 @@ class SentryTimberTree(
         message: String?,
         vararg args: Any?
     ) {
-        super.d(message, args)
+        super.d(message, *args)
         logWithSentry(Log.INFO, null, message, *args)
     }
 
@@ -92,7 +92,7 @@ class SentryTimberTree(
         message: String?,
         vararg args: Any?
     ) {
-        super.i(t, message, args)
+        super.i(t, message, *args)
         logWithSentry(Log.INFO, t, message, *args)
     }
 
@@ -107,7 +107,7 @@ class SentryTimberTree(
         message: String?,
         vararg args: Any?
     ) {
-        super.w(message, args)
+        super.w(message, *args)
         logWithSentry(Log.WARN, null, message, *args)
     }
 

--- a/sentry-android-timber/src/test/java/io/sentry/android/timber/SentryTimberTreeTest.kt
+++ b/sentry-android-timber/src/test/java/io/sentry/android/timber/SentryTimberTreeTest.kt
@@ -10,6 +10,7 @@ import io.sentry.IHub
 import io.sentry.SentryLevel
 import io.sentry.getExc
 import timber.log.Timber
+import java.util.MissingFormatArgumentException
 import kotlin.test.BeforeTest
 import kotlin.test.Test
 import kotlin.test.assertEquals
@@ -275,5 +276,11 @@ class SentryTimberTreeTest {
         val sut = fixture.getSut()
         sut.e(Throwable())
         verify(fixture.hub, never()).addBreadcrumb(any<Breadcrumb>())
+    }
+
+    @Test
+    fun `Tree does not throw when using log with args`() {
+        val sut = fixture.getSut()
+        sut.d("test %s, %s", 1, 1);
     }
 }

--- a/sentry-android-timber/src/test/java/io/sentry/android/timber/SentryTimberTreeTest.kt
+++ b/sentry-android-timber/src/test/java/io/sentry/android/timber/SentryTimberTreeTest.kt
@@ -10,7 +10,6 @@ import io.sentry.IHub
 import io.sentry.SentryLevel
 import io.sentry.getExc
 import timber.log.Timber
-import java.util.MissingFormatArgumentException
 import kotlin.test.BeforeTest
 import kotlin.test.Test
 import kotlin.test.assertEquals
@@ -281,6 +280,6 @@ class SentryTimberTreeTest {
     @Test
     fun `Tree does not throw when using log with args`() {
         val sut = fixture.getSut()
-        sut.d("test %s, %s", 1, 1);
+        sut.d("test %s, %s", 1, 1)
     }
 }

